pkg/asset/store.go

```diff
@@ -1,12 +1,20 @@
 package asset
 
 import (
+	"encoding/json"
+	"github.com/pkg/errors"
+	"io/ioutil"
+	"os"
+	"path/filepath"
 	"reflect"
 
-	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	stateFileName = ".asset_state.json"
```

Comment:
if somebody looks at this file, they wouln't know it belongs to openshift install.